exercise/templatetags/exercise.py

```diff
@@ -278,6 +279,17 @@ def get_format_info(format):
     except KeyError as e:
         raise RuntimeError('Invalid format: \'{}\''.format(format)) from e
 
+
 @register.simple_tag
 def get_format_info_list(formats):
     return [get_format_info(format) for format in formats.split()]
+
+
+def is_accessible(entry, context, t):
```

Comment:
Let's name the function `_is_accessible` and move it to the top of the file since the other helper functions are defined there. The underscore in the beginning implies that the function is not intended to be used outside this module (private function).

Reply:
Renamed.